ktor-utils/common/src/io/ktor/util/collections/ConcurrentList.kt

```diff
@@ -12,7 +12,7 @@ import kotlinx.atomicfu.locks.*
 
 private const val INITIAL_CAPACITY = 32
 
-@KtorExperimentalAPI
+@Deprecated("This is going to become internal.")
 public class ConcurrentList<T> : MutableList<T> {
```

Comment:
I'm not sure about that. There is no replacement for external users

Reply:
Reverted